metricbeat/module/ceph/monitor_health/monitor_health_test.go

```diff
@@ -36,7 +36,8 @@ func TestFetchEventContents(t *testing.T) {
 	absPath, err := filepath.Abs("../_meta/testdata/")
 	assert.NoError(t, err)
 
-	response, err := ioutil.ReadFile(absPath + "/sample_response.json")
+	response, err := os.ReadFile(absPath + "/sample_response.json")
+	assert.NoError(t, err)
```

Comment:
It's better to use `require` here as it will immediately fail the test. If we fail to read the file, there  is no point continuing with the test.
```suggestion
	require.NoError(t, err)
```

Reply:
What is the difference to line 37 `assert.NoError(t, err)` ? Why should we use here `require` instead of `assert`?

Reply:
`require` calls `t.Fatal` or `t.FailNow` which effectively interrupts the test right when the assertion fails. In that case if we continue the test after `os.ReadFile` fails, the test will only produce more errors that are not helpful, hence stopping the test when the file cannot be read is the best approach in my opinion.

That is the only comment/change I consider a blocker, and even that is not a hard blocker.

Reply:
> That is the only comment/change I consider a blocker, and even that is not a hard blocker.

I can't follow why this is considered a blocker. Before this change `err` was never checked before and the discussion `assert` vs `require` is a style discussion, that should not affect this PR.